pkg/util/runtime/features.go

```diff
@@ -110,11 +110,11 @@ var (
 		FeatureSDKGracefulTermination: true,
 		FeatureStateAllocationFilter:  true,
 		FeaturePodHostname:            true,
+		FeatureResetMetricsOnDelete:   true,
```

Comment:
can you fix the ordering here as well? It should be alphabetical within the stage. Looks like `PodHostname` also violates right now.

Reply:
rearranged the order